src/python/bot/tasks/fuzz_task.py

```diff
@@ -1900,12 +1900,19 @@ def run(self):
     # apply some of the time. Adjust APP_ARGS for them if needed.
     trials.setup_additional_args_for_app()
 
+    engine_impl = engine.get(self.fuzzer.name)
+
     # TODO(mbarbella): Remove the environment variable check when the old
     # blackbox pipeline has been converted to the engine interface.
-    engine_impl = engine.get(self.fuzzer.name)
     if not engine_impl and environment.get_value('USE_BLACKBOX_ENGINE'):
       engine_impl = engine.get('blackbox')
 
+    # TODO(mbarbella): Remove this check once AFL is fully converted to the new
+    # engine interface.
+    if (self.fuzzer.name == 'afl' and
```

Comment:
I'm guessing the plan is to turn this on for reproduction too once we verify that fuzzing seems to work?

Reply:
We should actually do this now. Made that change as well. It'll be gated behind a separate job type initially, so there's no harm in enabling in both cases.

Reply:
Ok, we need to be very careful here to make sure existing testcases don't break. Please verify this manually on a number of testcases after landing (prior to our progression cron).